### ISSUE TITLE
Allow login via email from /wp-admin route

### ISSUE THREAD

**Issue by claytoncollie** [MrApplejuice/wp-autologin-links#15]
My initial thought on what this plugin would do is a bit different than what is going on. I see the login link on the user profile and also in the admin bar on the frontend. All well and good. 

Am I correct in assuming that the administrator would manually generate the login link and then manually send that link via email or whatever to the client/user?

I thought this plugin would either a) hide the password field on the login screen b) provide another button the login screen to login via email. In either case, I thought that a user would go to /wp-admin, type in their email (they are already a user on the site) and then on submit, wp_mail would send the login link to their email address entered or to the email address in the user profile.  

Am I missing something or is this not the intention on the plugin?

**Comment by MrApplejuice** [comment]
> Am I correct in assuming that the administrator would manually generate the login link and then manually send that link via email or whatever to the client/user?

Yes, that is the idea.

> In either case, I thought that a user would go to /wp-admin, type in their email (they are already a user on the site) and then on submit, wp_mail would send the login link to their email address entered or to the email address in the user profile.

Well. That was not the original intention, no. But I got quite a stack of PMs now which all asked me for a (paid) feature like this. I made one, and I am thoroughly testing it right now but I find it a tad crappy which is the reason why this feature might make it into the base plugin at some time when I replace it with a better emailing feature.

With all other non-admin requests, there is the problem that this plugin actually circumvents the regular login mechanisms provided by wordpress. At the beginning, I had to jump a few hoops to get this through review at all, and to honor the (correctly cautious) complaints from the wordpress plugin reviewers, I will likely never make it possible for common users to generate an autologin link for themselves in the base-plugin. So this will never happen:

> I thought that a user would go to /wp-admin, type in their email

But instead (in the draft for the paid addon to this addon) the administrator can activate automated login creation on user registration. The user then gets a user-login code sent to their email address which they can use as an alternative to login to the website.

However, I am working on this in the evening so progress is not super-fast. It might take some time until this makes it into the "open/free" plugin.
